artifact-management/commands/gem-test.yml

```diff
@@ -1,36 +1,16 @@
-description: Perform the actual gem build
+description: Perform simple gem test
 parameters:
-  bundler_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
-    type: string
-    default: 1.17.3
-  gem_repo_host:
-    description: The host to which built packages should be pushed. 
-    type: env_var_name
-    default: AVVO_GEM_REPO_HOST
-  gem_repo_key:
-    description: The :rubygems_api_key used to access the gem repository host
-    type: env_var_name
-    default: AVVO_GEM_REPO_TOKEN
   gem_test_dir:
     description: The directory in which the gem tesets should run
     type: string
     default: .
-  packagecloud_read_token:
-    description: The token to be used when fetching packages from Packagecloud.
-    type: env_var_name
-    default: PACKAGECLOUD_READ_TOKEN
   test_command:
+    default: bundle exec rake test
     description: The command that should be used to run tests
     type: string
-    default: bundle exec rake test
 steps:
   - setup-gem-environment
-  - run:
-      name: Install bundler gem
-      command: |-
-        cd <<parameters.gem_test_dir>>
-        gem install bundler --version=<<parameters.bundler_version>>
+  - gem-test-common-steps
```

Comment:
TODO: Figure out how to pass the `bundler_version` that should be specifiable in `gem-test` into the `gem-test-common-steps` subcommand.